Rules.mk

```diff
@@ -118,7 +118,7 @@ help:
 	@echo '  all          - print this help message'
 	@echo '  build        - Build binary at ./cmd/ipfs/ipfs'
 	@echo '  nofuse       - Build binary with no fuse support'
-	@echo '  install      - Build binary and install into $$GOPATH/bin'
+	@echo '  install      - Build binary and install into $$GOBIN/bin'
```

Comment:
`$GOBIN` is `$GOPATH/bin` your change should be `$GOBIN` because here it would expend to `$GOPATH/bin/bin`

I don't fully see why someone would care bout `$GOPATH/bin` vs `$GOBIN`.
They should be the same everywhere, no ?